R/Simulations-methods.R

```diff
@@ -839,6 +844,24 @@ setMethod("show",
         "simulations\n\n"
       )
 
+      # report stopping rules
+      logPercent <- apply(object@stop_report, 2, mean) * 100
+
+
+
+      if (!is.null(names(logPercent))) {
+        cat("Stopping rules:\n \n")
+        for (i in 1:ncol(object@stop_report)) {
+          if (!is.na(names(logPercent)[i])) {
```

Comment:
this can be written more compact. Maybe first determine the indices for which the name is not NA. Also I do not see how it may happen that we have `NA` names.....?

Reply:
Updated - the logic is now: If there is ANY column containing no <NA> the Stopping rule reporting is initiated. For all non-<NA> columns the label and the logPercentage is output. The presence of <NA> or non-<NA> is the switch to output Stopping Rule reporting.

Reply:
Ok, then this code is more compact I think:
```
# Report individual stopping rules with non-<NA> labels.
stop_pct <- colMeans(object@stop_report) * 100
stop_pct_to_print <- stop_pct[!is.na(names(stop_pct))]
if (length(stop_pct_to_print) > 0) {
  cat(
    "Stopping rules:\n \n",
    paste(names(stop_pct_to_print), ": ", stop_pct_to_print, "%\n \n")
  )
}
```

Reply:
Oh wow - that makes a lot of sense 💯 Thank you very much!